test/TensorFlowNET.UnitTest/PythonTest.cs

```diff
@@ -196,17 +245,25 @@ public T evaluate<T>(Tensor tensor)
             //    return self._eval_helper(tensors)
             //  else:
             {
-                var sess = tf.Session();
+                var sess = tf.get_default_session();
                 var ndarray = tensor.eval(sess);
```

Comment:
@Wanglongzhi2001 what is the reason `tensor.eval` used there?
Maybe replace by `sess.run(tensor)` how it is made in original tests?

Reply:
I think it's to map the tensor to a node in the session (graph).